packages/core/src/features/st-var.ts

```diff
@@ -112,6 +120,61 @@ export function get(meta: StylableMeta, name: string): VarSymbol | undefined {
     return STSymbol.get(meta, name, `var`);
 }
 
+// Stylable StVar Public APIs
+
+export class StylablePublicApi {
```

Comment:
I don't like the Public in the name of this class. If it's exported it's public.